phpunit.xml

```diff
@@ -0,0 +1,22 @@
+<phpunit
```

Comment:
I'm sorry and It;s my fault. The [phpunit.xml](https://github.com/Th3Mouk/YahooWeatherAPI/blob/master/phpunit.xml) link is here.